.gitignore

```diff
@@ -12,3 +12,6 @@ build/
 .hgignore
 .venv/
 bemani.egg-info/
+server/
```

Comment:
What is this?

Reply:
I just store all the private server files here
```
~/bemaniutils/server# ls
api.sock  bemani         frontend.wsgi  install  preload    scores_tsvs  services.sock  uwsgi
api.wsgi  frontend.sock  game_data      nginx    scheduler  server.yaml  services.wsgi```